docs/src/howto/install/lxd.md

```diff
@@ -0,0 +1,196 @@
+# Install Canonical Kubernetes in LXD
+
+Canonical Kubernetes can also be installed inside an LXD container. This is a
+great way, for example, to test out clustered Canonical Kubernetes without the
+need for multiple physical hosts.
+
+## Installing LXD 
+
+You can install [LXD] via snaps:
+
+```no-highlight
+sudo snap install lxd
+sudo lxd init
+```
+
+## Add the Canonical Kubernetes LXD profile
+
+Canonical Kubernetes requires some specific settings to work within LXD (these
+are explained in more detail below). These can be applied using a custom
+profile. The first step is to create a new profile:
+```no-highlight
+lxc profile create k8s
+```
+
+Once created, we’ll need to add the rules. 
+
+```no-highlight
+wget https://raw.githubusercontent.com/canonical/k8s-snap/main/tests/e2e/lxd-profile.yaml -O k8s.profile
+```
+
+We can now pipe that file into the LXD profile.
+
+```no-highlight
+cat k8s.profile | lxc profile edit k8s
+```
+
+And then clean up.
+
+```no-highlight
+rm k8s.profile
+```
+
+## Start an LXD container for Canonical Kubernetes
+We can now create the container that Canonical Kubernetes will run in.
+
+```no-highlight
+lxc launch -p default -p k8s ubuntu:22.04 k8s
+```
+
+Note that this command uses the `default` profile, for any existing system
+settings (networking, storage, etc.) before also applying the `k8s` profile -
+the order is important.
+
+## Install Canonical Kubernetes in an LXD container
+First, we’ll need to install Canonical Kubernetes within the container.
+
+```no-highlight
+lxc exec k8s -- sudo snap install k8s --classic
+```
+
+## Access Canonical Kubernetes services within LXD
+Assuming you left the default bridged networking when you initially setup LXD,
```

Comment:
Briefly add a bit of explanation/review for the user what default bridged networking does.